agent/php_newrelic.c

```diff
@@ -327,13 +338,12 @@ static zend_function_entry newrelic_functions[] = {
     PHP_FE(newrelic_accept_distributed_trace_headers, newrelic_accept_distributed_trace_headers_arginfo)
     PHP_FE(newrelic_accept_distributed_trace_payload, newrelic_accept_distributed_trace_payload_arginfo)
     PHP_FE(newrelic_accept_distributed_trace_payload_httpsafe, newrelic_accept_distributed_trace_payload_httpsafe_arginfo)
+    PHP_FE(newrelic_get_request_metadata, newrelic_get_request_metadata_arginfo)
```

Comment:
Could you explain why this was pulled from the #ifdef PHP8 conditional compiliation?

Reply:
The definition was split between [PHP 8](https://github.com/newrelic/newrelic-php-agent/pull/755/files/c9a23d7759f896b61b6bfec224605876106e74df#diff-d585ba251eeaf9be978fe4a1557e1c03413315cc7d17267c4f6e86bca93b6ca2L332) `PHP_FE(newrelic_get_request_metadata, newrelic_arginfo_void)` and [phpNOT8](https://github.com/newrelic/newrelic-php-agent/pull/755/files/c9a23d7759f896b61b6bfec224605876106e74df#diff-d585ba251eeaf9be978fe4a1557e1c03413315cc7d17267c4f6e86bca93b6ca2L336) `PHP_FE(newrelic_get_request_metadata, 0)` because PHP 8 didn't allow void arginfo to be passed like the PHPNOT8 anymore.  Now that it has a non-void arginfo there's no need to split it, so it can be declared like all the other PHP_FEs that didn't have void arginfo.